### PR TITLE
LM-134: Add File Storage metrics

### PR DESCRIPTION
**Pull request by MarkvanMents**
Replacing #3997  in new Docs Stack structure

### CODE REVIEW

File: content/en/docs/developerportal/operate/metrics/trends-v4.md [hunk 1]
```diff
@@ -237,6 +234,22 @@ This graph should be interpreted in combination with other graphs. See [Combinin
 
 The disk usage graph shows only the disk usage inside the container. This is usually only relevant if your application creates a lot of temporary files in `/tmp`. This value is not the same as the file document storage.
 
+### 4.11 Application Number of Files{#Trends-appnumberoffiles}
+
+The **Application Number of Files** graph shows the number of files created by entities which are based on `FileDocument` generalizations, that are stored in S3 file storage.
+
+{{< figure src="/attachments/developerportal/operate/metrics/trends-v4/app-number-of-files.png" >}}
```
Comment (MarkvanMents) [inline content/en/docs/developerportal/operate/metrics/trends-v4.md:241]: Image missing from PR

File: content/en/docs/developerportal/operate/metrics/trends-v4.md [hunk 2]
```diff
@@ -237,6 +234,22 @@ This graph should be interpreted in combination with other graphs. See [Combinin
...
+
+### 4.12 Application Size of Files (in bytes){#Trends-appsizeoffiles}
+
+{{% alert type="info" %}}
+This metric was added on DATE to the [Cloud Foundry Mendix Buildpack](https://github.com/mendix/cf-mendix-buildpack). The data will only be visible once you deploy your Mendix app after that date. Before you redeploy, the graph will show **No Data Available**. After the deployment, data will only be shown on the graph from that date onward.
+{{% /alert %}}
+
+The **Application Size of Files** graph shows the size of files (in bytes) that are stored in file storage.
+
+{{< figure src="/attachments/developerportal/operate/metrics/trends-v4/app-size-of-files.png" >}}
```
Comment (MarkvanMents) [inline content/en/docs/developerportal/operate/metrics/trends-v4.md:251]: Image missing from PR

File: content/en/docs/developerportal/operate/metrics/trends-v4.md (hunk 1 above)
Comment (ajinkya-takle) [reply content/en/docs/developerportal/operate/metrics/trends-v4.md:241]: no_of_files

File: content/en/docs/developerportal/operate/metrics/trends-v4.md (hunk 2 above)
Comment (ajinkya-takle) [reply content/en/docs/developerportal/operate/metrics/trends-v4.md:251]: size_of_files

**Comment by MarkvanMents** [comment]
@ajinkya-takle 
I've added the images and updates the buildpack date as discussed.
Let me know when this can be released.

**Comment by udh-mendix** [comment]
@jeohist can you review and approve this PR? thanks!

**Review by jeohist — APPROVED** [review]
LGTM

**Pull request merged**